docs/gettinginvolved/acceptance-testing.rst

```diff
@@ -204,9 +204,21 @@ Managed
 ~~~~~~~
 
 You can also run acceptance tests on existing "managed" nodes.
-In this case the configuration file should include:
 
-- **addresses**: The IP addresses of two running nodes.
+This is a quicker way to run the acceptance tests as it avoids the slow process of provisioning new acceptance testing nodes.
+
+The ``managed`` provider:
+
+* re-installs the ``clusterhq-flocker-node`` and ``clusterhq-python-flocker`` packages,
+* creates and distributes new certificates and keys to all the nodes,
+* restarts the ``flocker-control`` on the first ``managed`` node, and
+* restarts the ``flocker-*-agent`` processes on all the nodes.
+
+This means that the ``managed`` provider can be used to quickly test different package versions and packages built from different branches.
+
+To use the ``managed`` provider, the configuration file should include:
+
+- **addresses**: A ``list`` of IP addresses of the nodes or a ``dict`` of ``{"<private_address>": "<public_address>"}`` if the public addresses are not configured on the node (see below).
```

Comment:
I thought about  forcing addresses to be a dict in all cases and just documenting that the hostname and private address should be same in the case of Rackspace....but that seemed ugly.

`{"1.2.3.4": "1.2.3.4"}`

I have a slight concern about ordering when using the dictionary form. With the list form, the first listed node is always the control node. With dicts that's not the case.